packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -1299,6 +1311,22 @@ class PopupMenuButton<T> extends StatefulWidget {
   /// Defaults to false.
   final bool useRootNavigator;
 
+  /// Used to override the default animation curves and durations of the popup
+  /// menu's open and close transitions.
+  ///
+  /// If [AnimationStyle.curve] is provided, it will be used to override
+  /// the default popup animation curve. Otherwise, defaults to [Curves.linear].
+  ///
+  /// If [AnimationStyle.reverseCurve] is provided, it will be used to
+  /// override the default popup animation reverse curve. Otherwise, defaults to
+  /// `Interval(0.0, 2.0 / 3.0)`.
+  ///
```

Comment:
Also mention:   `/// To disable the theme animation, use [AnimationStyle.noAnimation].`